Google Play Music.sln

```diff
@@ -17,8 +17,8 @@ Global
 		Release|x86 = Release|x86
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{B2C52D42-4EEE-445C-AD6D-357D93397937}.Debug|Any CPU.ActiveCfg = Release|x86
```

Comment:
For future reference I don't like config changes like this coming in on a feature change.  It just doesn't look right.  We don't even use the "Any CPU" config because CEFSharp requires a specific platform target.

Leave it in for now as it doesn't do any harm, just an FYI :smile_cat: